### PR TITLE
Maps find location (WIP)

### PR DESCRIPTION
**Pull request by kester-ng**
- Limited Maps location to only contains himalayan regions. (done)
- Finding location (done)
- Tests (done)

### CODE REVIEW

**Review by TanLeYang — COMMENTED** [review]
apart from adding the new env var into `project.yml`, looks good!

File: Challo/Challo/Modules/Map/MapModule.swift [hunk 1]
```diff
@@ -9,9 +9,22 @@ import SwiftUI
...
+                                mapRouteRepository: MapRouteRepository(),
+                                mapItineraryRepository: MapItineraryRepository())
+        
+        // TODO make url for alamofiremanager injected in
+        let networkManager = APINetwork.getNetworkManager()
+        if let alamofireManager = networkManager as? AlamofireManager {
+            alamofireManager.apiPath = ProcessInfo.processInfo.environment["google_places_path"]
+        }
+        
+        let placesAPI = PlacesAPI(parser: PlacesAPIParser(), networkManager: networkManager)
```
Comment (TanLeYang) [inline Challo/Challo/Modules/Map/MapModule.swift:22]: I think now we register the APIs with the SWInject `APIContainer` and then inject the APIs into the different modules in `MainContainerRouter` right?

File: Challo/Challo/Modules/Map/MapModule.swift [hunk 2]
```diff
@@ -9,9 +9,22 @@ import SwiftUI
...
     func assemble() -> (view: AnyView, presenter: MapPresenter) {
-        let interactor = MapInteractor()
+        let mapStore = MapStore(mapMarkerRepository: MapMarkerRepository(),
+                                mapRouteRepository: MapRouteRepository(),
+                                mapItineraryRepository: MapItineraryRepository())
+        
+        // TODO make url for alamofiremanager injected in
+        let networkManager = APINetwork.getNetworkManager()
+        if let alamofireManager = networkManager as? AlamofireManager {
+            alamofireManager.apiPath = ProcessInfo.processInfo.environment["google_places_path"]
```
[the remark this answers is not in the document]
Comment (kester-ng) [reply Challo/Challo/Modules/Map/MapModule.swift:19]: okay lemme add it in

File: Challo/Challo/Modules/Map/MapModule.swift (hunk 1 above)
Comment (kester-ng) [reply Challo/Challo/Modules/Map/MapModule.swift:22]: okay let me change it!

**Review by TanLeYang — APPROVED** [review]

**Pull request merged**